arclightning_frontend/games.html

```diff
@@ -53,6 +73,62 @@ <h1>Arc Lightning</h1>
 
 	]
             },
+
+	methods: {
+		sendSelectedGame: function() {
+			//check to see if game exists
+			var gameSelected = this.items[this.selectedIndex].exe_path
+			if(gameSelected)
+				axios.post('http://localhost:8000', {
+					id:gameSelected
+			})
+			/*debugging stuff*/
+			/*.then(function(response){
+				alert(response);
+			})
+			.catch(function (error) {
+				alert(gameSelected);
+			}) */
+			
+			
+
+		/* ajax functionality in case we dont end up using axios
+	
+			if (gameSelected)
+				$.ajax({ 
+					type: 'POST',
+					url: 'http://localhost:8080',
+					//sends path to executable for selected game
+					data: {id: gameSelected},
+				}) */
+		},
+
+		onKeyDown(e){
```

Comment:
is this the same function syntax as `document.addEventListener`? Let's rename `e` to `event` to be more clear